lib/noticed/base.rb

```diff
@@ -113,6 +113,7 @@ def run_delivery_method(delivery_method, recipient:, enqueue:, record:)
 
         # If the queue is `nil`, ActiveJob will use a default queue name.
         queue = delivery_method.dig(:options, :queue)
+        queue = send(queue) if queue.is_a? Symbol
```

Comment:
If users are already passing in the queue as a Symbol, this would be a breaking change. I think that's pretty common with ActiveJob because their docs show `queue_as :feeds`.

A better solution might be to accept a lambda or Proc and check if queue responds to `call`?

Reply:
I would have to test again, but I think this portion of noticed currently breaks if you pass in a symbol.

However, yes, I agree, with ActiveJob using symbols this might better be checked as you said, with `respond_to?(:call)`.

Let me make the changes and do some testing 👍

Reply:
Sounds like we should add some test cases too then. 😜

Reply:
Sounds good, i will bake tests in too.

Give me a few weeks on this one. Have family stuff next two weeks, so my time might be sparse when I am at the computer

Reply:
No worries! I might work on this myself in the meantime.

Reply:
Made the talked about changes. I also added tests both supporting that procs work and symbol names stay as strings 👍

Reply:
Excellent!

Reply:
Just pushed the typo error